using_images/s2i_images/customizing_s2i_images.adoc

```diff
@@ -69,10 +70,9 @@ The example shows a custom *_assemble_* script that prints the message,
 executes standard *_assemble_* script from the image and prints another message
 depending on the exit code of the *_assemble_* script.
 
-Wrapping the *_run_* script is a bit tricky because we have to use `exec` for
-invoking it (see chapter
-xref:../../creating_images/guidelines.adoc#general-docker-guidelines[Use `exec` in
-Wrapper Scripts]). As a consequence, it is impossible to have a post-run step
+Wrapping the *_run_* script is a bit tricky because you have to
+xref:../../creating_images/guidelines.adoc#general-docker-guidelines[use `exec`
+for invoking it]. As a consequence, it is impossible to have a post-run step
 because the code after `exec /usr/libexec/s2i/run` never gets executed.
```

Comment:
The above is verging on colloquialism and...kinda confusing. Maybe:

"You may have difficulties wrapping the *_run_* script using the `exec` command. As a consequence, a post-run step will also impossible(?), because..."

I use (?) because one thing being tricky leads to another being impossible? That's the confusing part I talk about...

cc: @bparees @php-coder